CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 1.1.0 / 2016-09-01
+
+* [CHANGE] Kubernetes SD: Drop `__meta_kubernetes_role`.
```

Comment:
Yup. That's the way (i.e. not mentioning it).

I missed that the commit description was referring to the example, not to the actual feature.